vds-api/src/lib.rs

```diff
@@ -15,115 +12,49 @@
 //!
 //! The following terminology is used in the VDS:
```

Comment:
this is pointless now, that there is only local and no more remote endpoints, no?